README.md

```diff
@@ -140,29 +142,35 @@ Available checker support
 
 #### Hunspell
 
-Requires a C++ compiler to compile the hunspell CXX source files which are part of `hunspell-sys`
+Requires a C++ compiler to compile the hunspell CXX source files which are part
+of `hunspell-sys`
 
 ##### Fedora 30+
+
 ```sh
 dnf install -y clang
 ```
 
 ##### Ubuntu 19.10+
+
 ```sh
 apt install -y clang
 ```
 
 ##### Mac OS X
-```
+
+```sh
 brew install llvm
 ```
 
-The environment variable `LLVM_CONFIG_PATH` needs to point to `llvm-config`, to do so:
+The environment variable `LLVM_CONFIG_PATH` needs to point to `llvm-config`, to
+do so:
```

Comment:
DId someone use reflow on the README? :heart:

Reply:
Yeah, but I did not realize it fucked up...

Reply:
And it worked nicely unlike my first comment suggested (phone renders of diffs are no good for that)